jsonparse.ps1

```diff
@@ -0,0 +1,83 @@
+$location=$args[0]
+$repo=$args[1]
+$name=$args[2]
+
+$location
+$repo
+$name
+$repo.Title
+$JSON = (Get-Content $location) -join "`n" | ConvertFrom-Json
+
+$base = "https://github.com/"
+if($repo.Title -eq "github")
+{
+  $base = "https://github.com/"
+}
+#Renaming the key value of file
```

Comment:
The `if` statement doesn't do anything. `$base` still ends up with the same value.